### PR TITLE
refactor: drop support for versions below Node.js 6

### PR DESCRIPTION
**Pull request by Siilwyn**

### CODE REVIEW

**Comment by ehmicky** [comment]
Thanks @Siilwyn for taking a hit at this!

A couple of notes:
  - [this](https://github.com/moxystudio/node-cross-spawn/blob/master/lib/util/readShebang.js#L15) can be removed too
  - according to #108, @satazor was suggesting dropping Node 6/7 as well

**Comment by satazor** [comment]
@Siilwyn This looks good!

I think we can remove support for node 6 as well. Its maintenance period ended in April.

Also, can you add a note on the readme saying to install cross-spawn@6 to support older node versions?

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,5 @@
 language: node_js
 node_js:
-  - '4.7'
-  - '4.8'
-  - '5.6'
-  - '5.7'
   - '6'
   - 'lts/*'
 # nodejs 11.11.X was failing with this https://travis-ci.org/moxystudio/node-proper-lockfile/jobs/503161746#L469
```
Comment (satazor) [inline .travis.yml]: Add ‘node’ to test latest stable node version, see https://github.com/moxystudio/stylelint-config-moxy/blob/cf536d49bb1ebf4e99d02ad764c107d8064c54f6/.travis.yml#L3

**Comment by Siilwyn** [comment]
:+1: I've changed to code following your feedback, please take another look!

**Comment by satazor** [comment]
Looks good! @ehmicky wanna give any final feedback?

**Comment by ehmicky** [comment]
Looks good to me, except one thing: shouldn't Travis run both for `node` (i.e. latest version) and `8`?

**Comment by satazor** [comment]
You are right, we need to define 8 explicitly in travis.yml.

**Comment by satazor** [comment]
Also, appveyor.yml needs to be updated accordingly.

**Comment by Siilwyn** [comment]
Thinking about this I would propose to be explicit for all tested versions, this way the CI doesn't suddenly break and it is easy to see which versions are tested. Plus it is easy to compare Travis and AppVeyor. Thoughts?

**Comment by ehmicky** [comment]
I think #123 can be closed accordingly.

**Review by Thompson1985 — APPROVED** [review]

**Pull request merged**